packages/dd-trace/src/profiling/tagger.js

```diff
@@ -22,7 +22,7 @@ const tagger = {
             .map(key => `${key}:${tags[key]}`))
         }
       case 'string':
-        return tagger.parse(tags.split(','))
+        return tagger.parse(tags.split(/[, ]+/))
```

Comment:
This introduces a bug because spaces are valid in tag values, which is not currently tested it seems. We need to make sure that the space is the last space before the next occurence of `:` to split only on the last space of a key-value pair.

Reply:
Thanks Roch I'll work on it!